src/libcore/ptr.rs

```diff
@@ -408,11 +408,12 @@ impl<T: ?Sized> *const T {
     ///
     /// Basic usage:
     ///
-    /// ```ignore
-    /// let val: *const u8 = &10u8 as *const u8;
+    /// ```
+    /// let val = 10u8;
+    /// let ptr: *const u8 = &val as *const u8;
```

Comment:
@estebank Reverted. Thought this was ` ```ignore `d because `&10u8` would be dropped too early (it was ` ```ignore `d to  avoid introducing a `#[feature]` to the example when `.as_ref()` was unstable).